### DISCUSSION TITLE
在b站看到的另一个绳网

### DISCUSSION THREAD

**Discussion by hjejdjdj** [share121/inter-knot#1276] [其他]
![Uploading Screenshot_20240811_113056_tv.danmaku.bili.jpg…]()
在b站的另一个绳网，这个绳网是真的吗。

**Comment by lllliufeng** [comment]
是直的。
还有图没传完啊xd

**Comment by hjejdjdj** [comment]
？

**Reply by lllliufeng** [reply]
是这样的，你要看Uploading Screenshot_20240811_113056_tv.danmaku.bili.jpg的Uploading消失才行

**Comment by cpu644** [comment]
哥们你图没传完

**Comment by hjejdjdj** [comment]
好吧谢谢提醒，可以删吗

**Reply by lllliufeng** [reply]
改就行了吧

**Comment by capacitor1** [comment]
> Reply::好吧谢谢提醒，可以删吗

可修改
